team/README.md

```diff
@@ -1,81 +1,89 @@
-# 👫 Team
+# <span class="emoji-bullet">👫</span> Team
 
 ![](../.gitbook/assets/neuropoly\_lab.jpeg)
 
 See [us on Github](https://github.com/orgs/neuropoly/people), or our more specific profiles below.
 
 ## Faculty
 
+||
+| - |
 | [Eva Alonso Ortiz](https://www.evaalonsoortiz.com) [![Twitter](../.gitbook/assets/logo\_twitter\_20px.png)](https://twitter.com/evaalonsoortiz) [![GitHub](../.gitbook/assets/logo\_github\_20px.png)](https://github.com/evaalonsoortiz)                                                                                                                                                                             |
-| --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | [Julien Cohen-Adad](faculty/julien-cohen-adad.md)  [![Twitter](../.gitbook/assets/logo\_twitter\_20px.png)](https://twitter.com/jcohenadad) [![GitHub](../.gitbook/assets/logo\_github\_20px.png)](https://github.com/jcohenadad/) [![LinkedIn](../.gitbook/assets/logo\_linkedin\_20px.png)](https://www.linkedin.com/in/jcohenadad/) [![E-Mail](../.gitbook/assets/logo\_mail\_20px.png)](mailto:jcohen@polymtl.ca) |
 | Nikola Stikov [![Twitter](../.gitbook/assets/logo\_twitter\_20px.png)](https://twitter.com/stikov)                                                                                                                                                                                                                                                                                                                    |
 
 ## Research Associates
 
+||
+| - |
```

Comment:
I think that seems reasonable! AFAIK, we had tables because we used to have another column for links to websites, etc.

But, if it's just a list of names (with icons), then I think using lists makes a lot of sense.

Reply:
Or uh we can just hide the empty headers with

```css
th { display: none }
```

In the medium-term, our choices are pretty flexible here. We could build a layout like https://www.tardiflab.com/team

![Screenshot 2021-11-03 at 12-38-47 Team — Tardif Lab](https://user-images.githubusercontent.com/987487/140103790-bbc4b191-377a-4626-bb36-73d6c187e6de.png)

**from** a list. A `<ul>` doesn't have to look like ` <ul>` in HTML: we can set styles like

```
#team li { display: inline-block } /* or maybe: display: flex */
#team li > img { ... } /* profile photos */
#team li h3 { ... } /* names */
#team li a[href="https://linkedin.com/*"] { ... } /* LinkedIn links */
#team li a[href="https://github.com/*"] { ... } /* Github links */
```

Reply:
Anyway I got rid of the empty headers, we can talk about photo layouts in #25